github/src/main/kotlin/gropius/Application.kt

```diff
@@ -20,6 +21,7 @@ class SyncApplication(
         runBlocking {
             incoming.sync()
         }
+        exitProcess(0);
```

Comment:
did you try closing the ApplicationContext instead?

Reply:
97e7d2bad00bc5df055901a6122ebc81ba380105